### PR TITLE
updated job documentation

### PR DESCRIPTION
**Pull request by shubham-shay-te**

### CODE REVIEW

File: website/docs/job.md [hunk 1]
```diff
@@ -36,9 +36,38 @@ def schedule(self, initial_delay: str, frequency: str, job: Job):
 def schedule_once(self, initial_delay: str, job: Job):
     ... 
 ````
```
Comment (shay-te) [inline website/docs/job.md:38]: please pull from massive-update. i added a log message upon job fail. 

add tests 
- run using `scheduler.schedule`  and see tests are running again and again 
- make a test useing `scheduler.schedule` when the job is failing, will it be runninfg again. 
- make a test useing `scheduler.schedule`, job will take 10 seconds to finish, 
   while will it is scheduled to run every 5 seconds. 
   what will happen then?  
   if it runs it is already running log a message that say already running. 
   make sure it is not running again.

File: website/docs/job.md [hunk 2]
```diff
@@ -36,9 +36,38 @@ def schedule(self, initial_delay: str, frequency: str, job: Job):
...
+job = UpdateCache()
+# updates the cache 1 second after the schedule_once() is called
+scheduler.schedule_once('1s', job)
+
+# updates the cache 1 second after the job is scheduled and keeps updating every 30 minutes until stop() is called.
+scheduler.schedule('1s', '30m', job) 
+
+scheduler.stop()# stops the ongoing job schedule 
+```
+
```
Comment (shay-te) [inline website/docs/job.md:70]: add  smalll example

**Review by shay-te — CHANGES_REQUESTED** [review]

File: tests/test_jobs.py [hunk 3]
```diff
@@ -46,3 +58,13 @@ def test_schedule(self):
         sleep(5)
         s.stop(j)
         self.assertGreater(j.called, 2)
+
+    def test_schedule_log(self):
+        s = JobScheduler()
```
Comment (shay-te) [inline tests/test_jobs.py:63]: the variable name `s`. what does this name mean?

File: tests/test_jobs.py [hunk 4]
```diff
@@ -46,3 +58,13 @@ def test_schedule(self):
...
+        job_exception = TestJobRaiseException()
+        with self.assertLogs() as cm:
+            s.schedule('1s', '1s', job_exception)
+            sleep(10.1)
+            log = str(cm.output)
+            self.assertIn('BaseException', log)
+            self.assertIn('Error while running job', log)
```
Comment (shay-te) [inline tests/test_jobs.py:70]: add another test please. where the job takes `10 seconds` to finish but it has a frequency of `5 seconds`
test how much time did it take it to run? 

are we running it again also when he is currently running?

thank you

Comment (shubham-shay-te) [reply tests/test_jobs.py:70]: I'll add the tests, I had kept the time low for the reason is we are using `sleep` to test the jobs, i had tried `freezegun` but it is failing here so it is consuming a lot of time just for a single test to run using `sleep`

Comment (shay-te) [reply tests/test_jobs.py:80]: let's resolve this on Thursday with the splash screen

**Review by shay-te — CHANGES_REQUESTED** [review]

File: tests/test_jobs.py (hunk 4 above)
Comment (shay-te) [reply tests/test_jobs.py:80]: lets remove this test

File: tests/test_jobs.py [hunk 5]
```diff
@@ -16,33 +16,56 @@ def run(self):
...
         sleep(2)
-        self.assertEqual(j.called, 1)
+        self.assertEqual(job.called, 1)
 
-        s.schedule('1s', '1s', j)
+        scheduler.schedule('1s', '1s', job)
         sleep(5)
-        s.stop(j)
-        self.assertGreater(j.called, 2)
+        scheduler.stop(job)
```
Comment (shay-te) [inline tests/test_jobs.py:59]: let test the stop actually happ[ened

**Review by shay-te — APPROVED** [review]

**Pull request merged**